contributing/checklist-command.md

```diff
@@ -17,6 +17,7 @@ CLI (command/) -> API Client (api/) -> HTTP API (command/agent) -> RPC (nomad/)
 *  [ ] Implement `-verbose` (expands truncated UUIDs, adds other detail)
 *  [ ] Update help text
 *  [ ] Implement and test new HTTP endpoint in `command/agent/<command>_endpoint.go`
+   * Register new URL paths in `command/agent/http.go`
```

Comment:
IMO makes sense to add this as a check item, bc a lot of tests won't necessarily won't go through our routing logic.